packages/wallets/src/features/cashier/modules/Transactions/Transactions.tsx

```diff
@@ -15,8 +15,9 @@ const filtersMapper: Record<string, Record<string, TFilterValue>> = {
     completed: {
         all: undefined,
         deposit: 'deposit',
-        transfer: 'transfer',
         withdrawal: 'withdrawal',
+        // eslint-disable-next-line sort-keys
+        transfer: 'transfer',
```

Comment:
lucky us living in times when browsers already respect the order in which keys are added to objects :D